### PR TITLE
[1LP][RFR]Add general UI test coverage

### PR DESCRIPTION
**Pull request by valaparthvi**
[](https://bugzilla.redhat.com/show_bug.cgi?id=)## Purpose or Intent
- __Adding tests__ :
    1. `test_red_hat_cloud_page_internal_server_error` for [BZ 1740131](https://bugzilla.redhat.com/show_bug.cgi?id=1740131)
    2. `test_provider_documentation` for [BZ 1741030](https://bugzilla.redhat.com/show_bug.cgi?id=1740130)
    3. `test_vm_right_size_recommendation_back_button` for [BZ 1733207](https://bugzilla.redhat.com/show_bug.cgi?id=1733207)
    4.  `test_compliance_column_header` for [BZ 1745660](https://bugzilla.redhat.com/show_bug.cgi?id=1745660)

### CODE REVIEW

File: cfme/tests/webui/test_general_ui_manual.py [hunk 1]
```diff
@@ -112,3 +112,105 @@ def test_pdf_summary_provider(provider):
         1503213
     """
     pass
+
+
+@pytest.mark.manual('manualonly')
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
```
Comment (john-dupuy) [inline cfme/tests/webui/test_general_ui_manual.py]: As in your other manual PR, I would add `startsin:` metadata for polarion.

File: cfme/tests/webui/test_general_ui_manual.py [hunk 2]
```diff
@@ -112,3 +112,105 @@ def test_pdf_summary_provider(provider):
...
+        testSteps:
+            1. Navigate to Red Hat Cloud > Providers
+        expectedResults:
+            1. There should be no 500 Internal Server Error
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
```
Comment (john-dupuy) [inline cfme/tests/webui/test_general_ui_manual.py]: same here ^^

File: cfme/tests/webui/test_general_ui_manual.py [hunk 3]
```diff
@@ -112,3 +112,105 @@ def test_pdf_summary_provider(provider):
...
+        caseimportance: low
+        initialEstimate: 1/18h
+        setup:
+            1. Take a fresh appliance with no provider
+        testSteps:
+            1. Log into the appliance.
+            2. Check where the link provided in `Learn more about this in the documentation.`
+                points to.
+        expectedResults:
+            1. Link must point to downstream documentation and not upstream.
```
Comment (john-dupuy) [inline cfme/tests/webui/test_general_ui_manual.py]: What is expected result 2.?

**Comment by valaparthvi** [comment]
Done with the changes @john-dupuy :)

File: cfme/tests/webui/test_general_ui_manual.py [hunk 4]
```diff
@@ -112,3 +112,106 @@ def test_pdf_summary_provider(provider):
         1503213
     """
     pass
+
+
+@pytest.mark.manual('manualonly')
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(coverage=[1740131])
+def test_red_hat_cloud_page_internal_server_error():
```
Comment (mshriver) [inline cfme/tests/webui/test_general_ui_manual.py]: @jawatts might want to remove this if your automation ends up covering it? Just a heads up for visibility.

**Pull request merged**